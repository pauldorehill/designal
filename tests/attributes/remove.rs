use designal::Designal;

#[derive(Designal)]
#[designal(remove)]
struct HumanBean();

#[derive(Designal)]
struct HumanBean1{
    #[designal(remove)]
    #[designal(remove)]
    taste: String
}

fn main() {}