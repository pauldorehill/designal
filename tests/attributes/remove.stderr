error: Remove is not valid at the container level
 --> $DIR/remove.rs:4:12
  |
4 | #[designal(remove)]
  |            ^^^^^^

error: You should only remove once
  --> $DIR/remove.rs:10:16
   |
10 |     #[designal(remove)]
   |                ^^^^^^
